scripts/set-availability.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This script adds the `available=true` field to all mentors that
+ * have not set this field already
+ * 
+ * Usage:
+ * 
+ *  $ yarn user:availability
+ * 
+ */
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+(async () => {
+  mongoose.connect(process.env.MONGO_DATABASE_URL, { useNewUrlParser: true });
```

Comment:
@crysfel The [mongoose.connect](https://mongoosejs.com/docs/api.html#mongoose_Mongoose-connect) function returns a Promise. Shouldn't we await a successful connection? 🤔 

```
await mongoose.connect(process.env.MONGO_DATABASE_URL, { useNewUrlParser: true });
```

/cc @brentmclark

Reply:
Seems reasonable to me.  Avoids a race condition.

Reply:
Good catch! 👌